libagent/formats.py

```diff
@@ -57,10 +59,39 @@ def parse_pubkey(blob):
 
     result = {'blob': blob, 'type': key_type, 'fingerprint': fp}
 
-    if key_type == SSH_NIST256_KEY_TYPE:
+    if key_type == SSH_NIST256_KEY_TYPE or key_type == SSH_NIST256_CERT_TYPE:
+        if key_type == SSH_NIST256_CERT_TYPE:
+            # nonce
+            _ = util.read_frame(s)
```

Comment:
Please remove the comment and use a dummy variable (here and below):
```suggestion
            _nonce = util.read_frame(s)
```

Reply:
Done as requested.

Reply:
Thanks!